src/plugins/Dashboard/FileCard.js

```diff
@@ -1,12 +1,27 @@
 const html = require('yo-yo')
 const getFileTypeIcon = require('./getFileTypeIcon')
+const onload = require('on-load')
 const { checkIcon } = require('./icons')
 
+let file
+const meta = {}
```

Comment:
Hmm, wouldn't this merge in edited metadata from previously edited files? If I change `a.png`'s "license" to "public domain" and then change `b.png`s "caption" to "hello world", `meta.license` would still be set to "public domain"

Reply:
😱